students/allen_maxwell/Lesson_03/list_lab.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+
+# Allen Maxwell
+# Python 210
+# 11/9/2019
+# list_lab.py
+
+# displays the list or indexed item in standard format
+def run_print(fruits, index=0):
+    menu=[]
+
+    # if no index is passed print full list
+    if index == 0:
+        for i in range(len(fruits)):
+            menu.append(print_format(i + 1, fruits[i]))
+
+    # else print indexed item
+    else:
+        menu = print_format(index, fruits[index - 1])
+    print(menu)
+
+# format the display item
+def print_format(index, item):
+    return str(index) + ') ' + item
+
+# standardize the list format
+def fruit_format(fruit):
+
+    # if fruit is not plural, pluralize it
+    if fruit.endswith('y'): 
+        fruit = fruit[0 : len(fruit) - 1]
+        fruit += 'ies'
+    elif fruit.endswith('ch'): 
+        fruit += 'es'
+    elif not (fruit.endswith('s') or fruit == ''):  
+        fruit += 's'
+
+    # if fruit is not capitalized, capitalize the first letters
+    fruit = fruit.title()
+    return fruit
```

Comment:
Love this function!  Have never seen a student pluralize user input in this exercise before - cool!